### PR TITLE
Kube postdeployment

### PR DESCRIPTION
**Pull request by devtrap**
- add some types for kubernetes for both master snapshot and snapshot configuration file 
- add traceback logger for kubernetes exceptions 
- add kubernetes documentation files

### CODE REVIEW

File: docs/docs/connectors/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "companyName": "Company Name",
+    "clusterName": "prancer-prod-prod-eastus2-aksprod01",
+    "clusterUrl": "https://prancer-kube:6443",
+    "namespaces": [
+        {
+            "namespace": "default",
+            "serviceAccounts": [
+                {
+                    "name": "prancer_ro",
+                    "secret":"eyJhbGciOiJSUzI1NiIsImtpZCI6ImpvS2RmQ3dfZ05URGY4TjRSZElvMno1OU9L"
```
Comment (farchide) [inline docs/docs/connectors/kubernetes.md]: remove the secret from the config file
just put a placeholder there

**Review by farchide — CHANGES_REQUESTED** [review]

**Review by ajeybk — APPROVED** [review]
Looks fine to me.

File: docs/docs/connectors/kubernetes.md (hunk 1 above)
Comment (devtrap) [reply docs/docs/connectors/kubernetes.md]: secret removed

**Review by vatsalgit5118 — APPROVED** [review]

File: docs/docs/connectors/kubernetes.md (hunk 1 above)
Comment (ajeybk) [reply docs/docs/connectors/kubernetes.md]: <secret token> place holder added.

**Review by farchide — APPROVED** [review]

**Pull request merged**